src/qibo/models/qcnn.py

```diff
@@ -42,12 +45,32 @@ class QuantumCNN:
 
     """
 
-    def __init__(self, nqubits, nlayers, nclasses=2, params=None):
+    def __init__(
+        self,
+        nqubits,
+        nlayers,
+        nclasses=2,
+        params=None,
+        twoqubitansatz=None,
```

Comment:
Can you tell me a bit more about this `twoqubitansatz` attribute?

Reply:
The qcnn has a two qubit convolutional layer which currently has a fixed architecture. The twoqubitansatz allows the user to change this convolutional layer into any two qubit circuit of their choice.